RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -518,40 +518,50 @@ void auto_scepterSkills()
 	{
 		return;
 	}
-	//Day 1 skills
-	if(my_daycount() == 1)
+	
+	if(canUse($skill[Aug. 24th: Waffle Day!]) && !get_property("_aug24Cast").to_boolean())
 	{
-		if(canUse($skill[Aug. 24th: Waffle Day!]) && !get_property("_aug24Cast").to_boolean())
-		{
-			use_skill($skill[Aug. 24th: Waffle Day!]); //get some waffles to hopefully change some bad monsters to better ones
-		}
-		if(canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean())
-		{
-			use_skill($skill[Aug. 30th: Beach Day!]); //Rollover adventures
-		}
-		if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && pathHasFamiliar() && ((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small()))
+		use_skill($skill[Aug. 24th: Waffle Day!]); //get some waffles to hopefully change some bad monsters to better ones
+	}
+	if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && pathHasFamiliar() && ((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small()))
+	{
+		if(!is100FamRun())
 		{
-			if(!is100FamRun())
-			{
-				use_familiar(findNonRockFamiliarInTerrarium()); //equip non-rock fam to ensure we get tiny gold medal
-			}
-			use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
+			use_familiar(findNonRockFamiliarInTerrarium()); //equip non-rock fam to ensure we get tiny gold medal
 		}
+		use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
+	}
+}
+
+void auto_scepterRollover()
+{
+	if(canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean() && get_property("_augSkillsCast").to_int()< 5)
```

Comment:
I see 2 options with baywatch
1. Keep here, but add check to confirm baywatch will actually be used for PJs
2. Put back in `auto_scepterSkills()` and always summon it like it was previously. Then can get the -2 mp skill benefit throughout the day. Mafia should auto equip when we cast stuff

Reply:
Going with Option 2. I think it's super unlikely that there's a user that would have pulled or can get 3 of the 5 accessories that tie or are better